cpp/tests/groupby/argmax_tests.cpp

```diff
@@ -118,6 +118,61 @@ TYPED_TEST(groupby_argmax_test, null_keys_and_values)
   test_single_agg(keys, vals, expect_keys, expect_vals, std::move(agg2), force_use_sort_impl::YES);
 }
 
+template <typename T>
+using FWCW = cudf::test::fixed_width_column_wrapper<T>;
+
+TYPED_TEST(groupby_argmax_test, structs)
```

Comment:
Struct is still not enabled in hash aggregation. So this test is just irrelevant.

Reply:
If you want to enable struct, do that `at can_use_hash_groupby`.

Reply:
`can_use_hash_groupby` determines whether we can use hash-based algorithms based on the data type of values but not keys. The test here is exercising struct-of-struct keys with integer values so the underlying algorithm is actually hash-based.

This PR is a preparation step to finally close #8039 and #10181 where the feature requests focus on supporting `LIST`/`STRUCT` keys. `LIST`/`STRUCT` values, if it's possible (haven't looked into details yet), should be targeted separately.

Reply:
So nevertheless this test will not run with hash-based aggregate. If enabling struct/list is beyond scope of this PR, let's remove this test and move it to the following PR.

Reply:
I believe tests for new types of keys go in `keys_tests.cpp`. The `<agg>_tests.cpp` files are for testing the aggregations

Reply:
The new benchmark is ready once https://github.com/rapidsai/rapids-cmake/pull/193 is merged.